Microsoft/microsoft-sentinel/ingest/parser.yml

```
@@ -0,0 +1,38 @@
+name: microsoft-sentinel
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+        output_field: message
+  - name: read_event_date
+    external:
+      name: date.parse
+      properties:
+        input_field: "{{json_event.message.alertTime}}"
+        output_field: datetime
+  - name: set-common-fields
+  - name: set-sentinel-fields
+
+stages:
+  set-common-fields:
+    actions:
+      - set:
+          "@timestamp": "{{read_event_date.datetime}}"
+          event.kind: "{{json_event.message.alertKind}}"
+          event.reason: "{{json_event.message.alertDescription}}"
+          observer.product: "{{ json_event.message.productName}}"
+          observer.vendor: "{{ json_event.message.vendorName}}"
+  set-sentinel-fields:
+    actions:
+      - set:
+          sentinel.alert.severity: "{{json_event.message.severity}}"
```

I suggest moving `AlertKind` to a custom field.
```suggestion
          sentinel.alert.severity: "{{json_event.message.severity}}"
          sentinel.alert.kind: "{{json_event.message.alertKind}}"
```